Workbooks/Workloads-AMA/Add monitoring virtual machine/Add monitoring virtual machine.workbook

```diff
@@ -469,7 +469,7 @@
                     "componentIdSource": "parameter",
                     "componentId": "ResourceGroup",
                     "templateUriSource": "static",
-                    "templateUri": "https://azmonsolutions.blob.core.windows.net/workloadinsights/add-monitoring-vm-azuredeploy.json",
+                    "templateUri": "ARMTemplates/AddMonitoringVirtualMachine.armtemplate",
```

Comment:
This should read something like 'Community-Workbooks/Workloads-AMA/Add monitoring virtual machine/AddMonitoringVirtualMachine.armtemplate'

Pretty much the path of the github repo, except the physical folder path 'workbooks' is 'Community-Workbooks'